mocha-reporter-config-plugins.json

```diff
@@ -0,0 +1,6 @@
+{
+  "reporterEnabled": "spec, mocha-junit-reporter",
```

Comment:
Is there a default path for the config file, or a way to set this without an additional config file?

Reply:
I wasn't able to set multiple reporters without this